### PR TITLE
Add Rallly self-hosted scheduling app

### PR DESCRIPTION
**Pull request by kirkins**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please check PR guidelines and check the boxes.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by kirkins** [comment]
Hi I'm having trouble figuring out which version of CC-BY-NC this project is.

According to the repo README it is CC-BY-NC but the list on awesome-selfhosted includes several versions. I'm not sure how to tell which one it is. Can anyone assist? 

https://github.com/lukevella/Rallly/blob/master/LICENSE.txt

**Comment by nodiscc** [comment]
@kirkins Thanks for the patch, the license text is from the [CC-BY-SA v3](https://creativecommons.org/licenses/by-sa/3.0/legalcode) license (v4 does not refer to "CCPL" anymore, v2 has "Fair use rights" section whereas v3 has "Fair Dealing Rights").

Awesome-Selfhosted currently only lists the v4 license. You could add a new `v3` license tags, a new licenses table entry... but instead I suggest we stop caring about the CC License versions (differences are [minor](https://creativecommons.org/share-your-work/licensing-considerations/version4/)) and just link to the latest available one; you could:

 * Rename all `CCBYSAv4` entries to just `CCBYSA`
 * Rename the licenses list entry to `Creative Commons Attribution-ShareAlike International License` (remove `4.0`)

And I would merge it.

**Comment by kirkins** [comment]
Does that look right?

File: README.md [hunk 1]
```diff
@@ -1206,11 +1206,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
  * `Beerware` - [Beerware License](https://spdx.org/licenses/Beerware.html)
  * `BSD` - [FreeBSD License](https://www.freebsd.org/copyright/license.html)
  * `BSD` - [BSD 2-Clause](https://opensource.org/licenses/BSD-3-Clause) or [BSD 3-Clause](https://opensource.org/licenses/BSD-3-Clause) license
- * `CCBYNCNDv4` - [Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International](https://creativecommons.org/licenses/by-nc-nd/4.0/)
- * `CCBYNCSAv2` - [Creative Commons Attribution-NonCommercial-ShareAlike 2.0](https://creativecommons.org/licenses/by-nc-sa/2.0/)
- * `CCBYNCSAv3` - [Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported](https://creativecommons.org/licenses/by-nc-sa/3.0/)
- * `CCBYNCv4` - [Creative Commons Attribution-NonCommercial 4.0 International](https://creativecommons.org/licenses/by-nc/4.0/)
- * `CCBYSAv4`  - [Creative Commons Attribution-ShareAlike 4.0 International License](https://creativecommons.org/licenses/by-sa/4.0/)
+ * `CCBYSA`  - [Creative Commons Attribution-ShareAlike International License](https://creativecommons.org/licenses/by-sa/4.0/)
```
Comment (nodiscc) [inline README.md]: Related: remove CCBYNC* items, tracked at https://github.com/Kickball/awesome-selfhosted/issues/983

**Comment by nodiscc** [comment]
Ok to merge, some more work to be done, tracked at https://github.com/Kickball/awesome-selfhosted/issues/983

**Comment by kirkins** [comment]
I thought the changes were approved so I deleted my fork. I made another pull request with the license changes https://github.com/Kickball/awesome-selfhosted/pull/984

**Comment by nodiscc** [comment]
Thank you

**Pull request merged**